Userland/Utilities/pro.cpp

```diff
@@ -233,11 +233,12 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     bool following_url = false;
```

Comment:
Commit message:

> There were previously `report_time_in_ms` and `speed_update_time_in_ms` variables

These variables still exist, so they are not "previously"

Reply:
Yeah, that wasn't very well worded. I've rewritten it to make it clearer what I meant.